### PR TITLE
Perform ctrl step and subsequent process only for specified scenarii

### LINKED ISSUE #46: Don't run the `ctrl` step for each scenarios
Currently, when at the `ctrl` step, we compute and output every scenario. However, we might already know which kind of scenario we want to run. As we discussed it earlier, it might be good to only compute scenarios that are defined in the parameter file.

The same idea should also be applied to the post-ctrl `process`, as it currently process every scenario and might throw errors if one of them is missing.

### LINKED ISSUE #96: process step creates html file for each scenario
Output created: /home/51_htmlFiles/05_process_Control_goodQualityCells_2024-03-26.html
[[1]]
[1] "/home/51_htmlFiles/05_process_Control_goodQualityCells_2024-03-26.html"

[[2]]
[1] "/home/51_htmlFiles/05_process_Control_goodQualityCells_2024-03-26.html"

- Process step creates html output for each scenario and overwrites but it`s supposed to create only one.

### PR DESCRIPTION
**Pull request by Rumengol**
And a few minor fixes.

Would resolve #46 and close #96

### CODE REVIEW

File: 02_process_pipeline.Rmd [hunk 1]
```diff
@@ -322,7 +294,7 @@ of the dataset. We also save the clustering information as a csv file.
 
 ```{r exports}
 # variable features
-saveHVG <- function(var.feats.dataset, scenario) {
+saveHVG <- function(var.feats.dataset) {
```
Comment (celinech) [inline 02_process_pipeline.Rmd:297]: useless function, as it only takes the var.feats.list as input, directly used as the object to write.

File: 04_additionalControls.Rmd [hunk 2]
```diff
@@ -349,8 +352,13 @@ rm(doubletsNb)
 invisible(gc())
 ```
 
-# Cell cycle regressions - scenarios 2 and 3
-
+```{r scenarios-2-3-title, echo=FALSE, results='asis'}
+if(scenarios==c("1")){
+  cat("# NOT RELEVANT")
```
Comment (celinech) [inline 04_additionalControls.Rmd:357]: You will create a title named **NOT RELEVANT** ??

**Review by celinech — COMMENTED** [review]
multiple comments to come over

File: 02_process_pipeline.Rmd [hunk 3]
```diff
@@ -142,26 +142,20 @@ if (goodQ) {
 # Load `r DATASET` data
 
 ```{r load-data-1, eval=!goodQ & !combinedD, echo=!goodQ & !combinedD}
-SO.list <- list(readRDS(paste0(PATH_RDS_OBJECTS, "/01_qc_", DATASET, "_", FILTER, ".rds")))
-names(SO.list) <- FILTER
+scenario = 1
```
[the remark this answers is not in the document]
Comment (Rumengol) [reply 02_process_pipeline.Rmd:145]: It is actually used at l.301
`    if (goodQ & scenario != "1") {`
In R, both conditions are tested in an AND clause, even if the first is false. Thus, the scenario variable must exists.

**Review by celinech — DISMISSED** [review]

**Review by celinech — APPROVED** [review]

**Review by sevdarafatov — APPROVED** [review]

**Pull request merged**